api/app/api.py

```diff
@@ -19,7 +41,105 @@
     allow_headers=["*"]
 )
 
+class ForecastRequest(BaseModel):
```

Comment:
Could you use this to a models.py file in the same dir

Reply:
> Could you use this to a models.py file in the same dir

yes sure

Reply:
> > Could you use this to a models.py file in the same dir
> 
> yes sure

done this in my recent commit